### PR TITLE
EFA-500 Fix "add account" button not displaying

### PR DESCRIPTION
**Pull request by ig-garcia**

### CODE REVIEW

File: k9mail/src/main/res/layout-land/accounts.xml [hunk 1]
```diff
@@ -1,115 +1,127 @@
...
-            android:paddingEnd="16dp"
-            android:text="@string/global_settings_title"
-            android:textColor="?attr/colorAccent" />
-
-        <FrameLayout
-            android:id="@+id/generalSettingsContainer"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            tools:layout_height="300dp" />
+        android:layout_marginTop="16dp"
```
Comment (pzplanck) [inline k9mail/src/main/res/layout-land/accounts.xml:19]: can we apply styles for padding, etc.?

File: k9mail/src/main/res/layout-land/accounts.xml [hunk 2]
```diff
@@ -1,115 +1,127 @@
...
+        android:layout_marginBottom="8dp"
```
Comment (pzplanck) [inline k9mail/src/main/res/layout-land/accounts.xml:20]: what is the purpose of 8dp, is that aligned with the Material design guidelines?

File: k9mail/src/main/res/layout-land/accounts.xml [hunk 3]
```diff
@@ -1,115 +1,127 @@
...
+        android:paddingEnd="16dp"
+        android:text="@string/available_accounts_title"
+        android:textColor="?attr/colorAccent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@id/separator"
+        app:layout_constraintTop_toBottomOf="@id/appbar" />
+
+    <ProgressBar
+        android:id="@+id/loading"
+        android:layout_width="64dp"
```
Comment (pzplanck) [inline k9mail/src/main/res/layout-land/accounts.xml:70]: style might be applied.
is this progress bar looks properly for all range of supported by app screens?

File: k9mail/src/main/res/layout-land/accounts.xml (hunk 2 above)
Comment (ig-garcia) [reply k9mail/src/main/res/layout-land/accounts.xml]: I am just keeping what we had before there, just refactored the layout but did not change any sizes etc.

**Review by pzplanck — APPROVED** [review]

**Pull request merged**